src/shapes/Text/Text.ts

```diff
@@ -1438,23 +1438,12 @@ export class Text<
    * @private
    */
   _clearCache() {
+    this._forceClearCache = false;
     this.__lineWidths = [];
     this.__lineHeights = [];
     this.__charBounds = [];
   }
 
-  /**
-   * @private
-   */
-  _shouldClearDimensionCache() {
-    const shouldClear = this._forceClearCache;
-    if (shouldClear) {
-      this.dirty = true;
-      this._forceClearCache = false;
-    }
-    return shouldClear;
-  }
```

Comment:
dismantled this method.
`dirty` is now set in `initDimensions`
`_forceClearCache` is reset in `_clearCache`

makes a lot more sense